.nycrc

```diff
@@ -5,7 +5,9 @@
     "text-summary"
   ],
   "include": [
-    "src/**/*.js"
+    "src/**/*.js",
+    "lib/**/*.js",
```

Comment:
Should lib really be added? The lib files are essentially the src files, so adding src should be enough?

Reply:
There is some src file in lib, not sure why, but therefore I believe we need to add lib.

Reply:
I’m not sure why we don’t have a `src` folder and have these files in it either, but I didn’t want to change the existing structure. Feel free to update it if you think it will be cleaner that way. For the record, I didn’t create the lib folder it was already here when I joined, and I had the same reaction as you! 😄